Frameworks/CoreFoundation/Locale.subproj/CFCalendar.c

```diff
@@ -746,6 +746,12 @@ static CFRange __CFCalendarGetRangeOfUnit2(CFCalendarRef calendar, CFCalendarUni
     // WINOBJC: Add cases for weekOf units. ICU will handle them below.
     case kCFCalendarUnitWeekOfYear:
     case kCFCalendarUnitWeekOfMonth:
+    // WINOBJC: Asking for the range of WeekOfMonth in Year always returns 6.
```

Comment:
Calculate provides the number of weeks within that particular month if the larger unit is the year. As you pointed out previously, this is incorrect. The behavior is to always return 6 weeks. Thus we can return 6 here instead of relying on calculate.

---
In reply to: [74336418](https://github.com/Microsoft/WinObjC/pull/844#discussion_r74336418) [](ancestors = 74336418)